Group 4 of Group 2 Review on 20200325

skjohansen opened the pull request.

murkug committed:
initial commit

@erifredrika @adgnascor @irvper77 @muratkugic: initial confused commit

murkug committed:
Added Database

@murakugic @erifredrika @adgnascor @irvper77 added database and relationships code first

Commit:
@adgnascor

Added my own environment and connectionstring

Commit:
@adgnascor

Added StarwarsShip request to API

First try, to figure out how it works

erifredrika committed:
added people api

Added classes and stuff so we can get information out of the api

erifredrika committed:
Merge branch 'Dev' of https://github.com/PGBSNH19/the-spaceport-group-2 into Dev

Adgnascor committed:
Delete NewFile2.txt

erifredrika committed:
added name

Name

erifredrika committed:
Added if/else and cool header

Added some logic, perhaps temporary. We could of course do it more neatly I guess.

Also added a super cool header yo

erifredrika committed:
Minor changes

Added toLower

erifredrika committed:
Updated database stuff

Updated database tables

erifredrika committed:
added a table

added table (everyone)

Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>
Co-Authored-By: Murat <muratkugic@users.noreply.github.com>

erifredrika committed:
hej

test merge

erifredrika committed:
Merge branch 'Fredrika' into Dev

murkug committed:
changed connectionstring

murkug committed:
spring cleaning

murkug committed:
trying to get figure out who parked where

murkug committed:
testing to see if my theory holds

murkug committed:
This seems to work satifactory..

I can now stear how many available spaces I have in the garage and it adjusts accordingly when it is full -> program exits.

murkug committed:
.

murkug committed:
Menu working

murkug committed:
improved menu

murkug committed:
cleaned up code

murkug committed:
I can now retrieve a name from the database who is currently occupying a parkinglot

murkug committed:
.

murkug committed:
spring cleaning before code review

@erifredrika @adgnascor @irvper77 @muratkugic

PontusGillenang commented on Source/SpacePark.Library/Context/SpaceParkContext.cs:
```diff
@@ -0,0 +1,20 @@
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(@"Data Source=DESKTOP-GO4PDLN;Database=SpaceParkDB;Trusted_Connection=True;");
```
Denna sök strängen skulle man kunna lägga i en separat json fil.

muratkugic replied:
Gjorde det :)

PontusGillenang commented on Source/SpacePark.Library/Migrations/20200323141422_wedidsomethingnew.cs:
```diff
@@ -0,0 +1,114 @@
+
+namespace SpacePark.Library.Migrations
+{
+    public partial class wedidsomethingnew : Migration
```
Kanske en bättre namngivning för att bättre veta vad som ändrats.

PontusGillenang commented on Source/SpacePark.Library/Migrations/20200323141758_wedidsomethingnewagain.cs:
```diff
@@ -0,0 +1,74 @@
+
+namespace SpacePark.Library.Migrations
+{
+    public partial class wedidsomethingnewagain : Migration
```
Även här skulle det vara bättre med en annan namngivning så man har bättre förståelse av vad som ändrats.

PontusGillenang commented on Source/SpacePark.Library/Migrations/20200323141929_wechangedback.cs:
```diff
@@ -0,0 +1,74 @@
+
+namespace SpacePark.Library.Migrations
+{
+    public partial class wechangedback : Migration
```
Bättre beskrivning här nu förstår man vad som hänt lite bättre, namngivningen är väl bättre att skriva i PascalCase.

PontusGillenang commented on Source/SpacePark.Library/Migrations/20200324103652_first.cs:
```diff
@@ -0,0 +1,17 @@
+        {
+
+        }
+    }
```
De tomma migrationerna ni har kan man antigen ta bort eller göra en roleback på för att gå tillbaka till det förra. Eftersom den är tom.

PontusGillenang reviewed (commented):
Bra skriven och strukturerad kod!

erifredrika committed:
delete update

delete update

erifredrika committed:
update

update

murkug committed:
.

murkug committed:
.

murkug committed:
.

erifredrika committed:
baseapp working

works, kind of

Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: Murat <muratkugic@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>

murkug committed:
Merge branch 'newnewDev' into Murat

Commit:
@adgnascor

Adgnascor committed:
Deleted appsettings

Commit:
@adgnascor

Test if appsetting.json still exist in repo

Commit:
@adgnascor  Added to GitIgnore

Adgnascor committed:
Delete appsettings.json

Adgnascor committed:
Update .gitignore

Commit:
@adgnascor  Fighting With GitIgnore

Commit:
Merge branch 'William_Dev' of https://github.com/PGBSNH19/the-spaceport-group-2 into William_Dev

Commit:
@adgnascor Removed comment

Commit:
@adgnascor  Fixed Payment problem

Fixed problem when trying to pay after relaunching the application. Before it crashed, now it work when entering correct name

murkug committed:
delete file since it only need to be on local

murkug committed:
deleted empty file

murkug committed:
.

Commit:
@adgnascor

Full Working (But simple) application

murkug committed:
.

murkug committed:
Merge branch 'master' into Murat

Commit:
@adgnascor

Trying to fix problem when entering wrong name or empty name!

murkug committed:
Merge branch 'master' into Murat

murkug committed:
error checking when entering name and ship

Co-Authored-By: Fredrika Eriksson <erifredrika@users.noreply.github.com>
Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>

murkug committed:
Added tests

Co-Authored-By: Fredrika Eriksson <erifredrika@users.noreply.github.com>
Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>
